src/styles/print.css

```
@@ -1,31 +1,182 @@
-/* AI 리포트 인쇄 — 드로어 패널 외 모든 요소 숨김 */
+/* AI 요약 보고서 인쇄·PDF — 화면에서는 숨김 */
+.ai-report-print-root {
+  position: absolute;
+  width: 0;
+  height: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)